docs: Add example on how to change configuration options in Kate (#15313)
rust-lang/rust-analyzer: docs/user/manual.adoc

> ----

Reviewer: We can probably drop this part now.

Then click on apply, and restart the LSP server for your rust project.

> To change rust-analyzer config options, the following example of JSON can be put into Kate's "User Server Settings" tab:

Reviewer: ```suggestion
To change rust-analyzer config options, start from the following example and put it into Kate's "User Server Settings" tab:
```
